### PR TITLE
Added Press kits to News

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
Added press kit for MeshMark Press Release. 

Screenshot 2022-05-27 at 3 20 49 AM

Press Kits for any other news articles/press releases can be added by:

1.  Creating the zip folder under `static/press-kits/`
2. Adding `presskit: "/press-kits/name.zip"` to the frontmatter of the individual news mdx files.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Will you make one of these two buttons the alt (green) color?

File: src/sections/Company/News-single/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <NewsSidebarWrapper>
+      {props.kit ? 
+        <a href={props.kit}>
+          <Button primary title="Get the Press Kit" external={true}>
+            <FiDownloadCloud size={21} className="icon-left" />
+          </Button>
+        </a> : "" }
+      <Link to="/brand">
+        <Button primary title="Check out our Brand Kit" external={true} />
```
Comment (leecalcote) [inline src/sections/Company/News-single/Sidebar.js:58]:
```suggestion
        <Button primary title="Layer5 brand kit" external={true} />
```

File: src/sections/Company/News-single/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+`;
+
+const NewsSidebar = (props) => {
+  return (
+    <NewsSidebarWrapper>
+      {props.kit ? 
+        <a href={props.kit}>
+          <Button primary title="Get the Press Kit" external={true}>
```
Comment (leecalcote) [inline src/sections/Company/News-single/Sidebar.js:53]:
```suggestion
          <Button primary title="Get the press kit" external={true}>
```

**Comment by l5io** [comment]
🚀 Preview for commit 108c198fbd06424b389e57982fc544d58019f0c4 at: https://628ff9773271d80daf3717b5--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Looking ahead, do you mind creating two levels of folders: `press/kits`?

File: src/sections/Company/News-single/Sidebar.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          </Button>
+        </a> : "" }
+      <Link to="/brand">
+        <Button primary title="Check out our Brand Kit" external={true} />
+      </Link>
+      <a href="mailto:press@layer5.io" target="_blank" rel="noreferrer">
+        <div className="project__block__inner">
+          <h5>Media Contacts</h5>
+          <p>Kate Suttons</p>
+          <FcSms size={40} className="icon" />
```
Comment (leecalcote) [inline src/sections/Company/News-single/Sidebar.js:64]: Given that this icon is a `mailto` link, let's switch to an envelope / email icon.

**Comment by leecalcote** [comment]
Great job making this new section responsive. 👍

**Review by warunicorn19 — APPROVED** [review]
Code wise LGTM!

File: src/sections/Company/News-single/Sidebar.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <NewsSidebarWrapper>
+      {props.kit ? 
+        <a href={props.kit}>
+          <Button primary title="Get the press kit" external={true}>
+            <FiDownloadCloud size={21} className="icon-left" />
+          </Button>
+        </a> : "" }
+      <Link to="/brand">
+        <Button secondary title="Check out our Brand Kit" external={true} />
```
Comment (debo19) [inline src/sections/Company/News-single/Sidebar.js:58]:
```suggestion
        <Button secondary title="Layer5 brand kit" external={true} />
```

**Comment by l5io** [comment]
🚀 Preview for commit fc9755a25cc3264e400c712573f6e24ffa803d63 at: https://629000868eb9ed1808f3df7c--layer5ng.netlify.app

**Review by leecalcote — COMMENTED** [review]
fit into one line

**Review by leecalcote — APPROVED** [review]

**Pull request merged**